sdk/python/foundation-models/system/inference/image-text-embeddings/image-text-embeddings-online-endpoint.ipynb

```diff
@@ -0,0 +1,487 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import base64\n",
+    "import json\n",
+    "\n",
+    "sample_image_1 = os.path.join(dataset_dir, \"milk_bottle\", \"99.jpg\")\n",
+    "sample_image_2 = os.path.join(dataset_dir, \"can\", \"1.jpg\")\n",
+    "labels = \"water_bottle, milk_bottle, carton, can\"\n",
+    "\n",
+    "\n",
+    "def read_image(image_path):\n",
+    "    with open(image_path, \"rb\") as f:\n",
+    "        return f.read()\n",
+    "\n",
+    "\n",
+    "request_json = {\n",
+    "    \"input_data\": {\n",
+    "        \"columns\": [\"image\", \"text\"],\n",
+    "        \"index\": [0, 1],\n",
+    "        \"data\": [\n",
+    "            [\n",
+    "                base64.encodebytes(read_image(sample_image_1)).decode(\"utf-8\"),\n",
+    "                \"\",\n",
+    "            ],  # the \"text\" column should contain empty string\n",
+    "            [base64.encodebytes(read_image(sample_image_2)).decode(\"utf-8\"), \"\"],\n",
+    "        ],\n",
+    "    }\n",
+    "}\n",
+    "\n",
+    "# Create request json\n",
+    "request_file_name = \"sample_request_data.json\"\n",
+    "with open(request_file_name, \"w\") as request_file:\n",
+    "    json.dump(request_json, request_file)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Score the sample_score.json file using the online endpoint with the azureml endpoint invoke method\n",
+    "response = workspace_ml_client.online_endpoints.invoke(\n",
+    "    endpoint_name=online_endpoint_name,\n",
+    "    deployment_name=demo_deployment.name,\n",
+    "    request_file=request_file_name,\n",
+    ")\n",
+    "print(f\"raw response: {response}\\n\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 5.2 Test the endpoint - text samples\n",
+    "\n",
+    "We will fetch some sample data from the test dataset and submit to online endpoint for inference."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import base64\n",
+    "import json\n",
+    "\n",
+    "sample_image_1 = os.path.join(dataset_dir, \"milk_bottle\", \"99.jpg\")\n",
+    "sample_image_2 = os.path.join(dataset_dir, \"can\", \"1.jpg\")\n",
+    "labels = \"water_bottle, milk_bottle, carton, can\"\n",
+    "\n",
+    "\n",
+    "def read_image(image_path):\n",
+    "    with open(image_path, \"rb\") as f:\n",
+    "        return f.read()\n",
+    "\n",
+    "\n",
+    "request_json = {\n",
+    "    \"input_data\": {\n",
+    "        \"columns\": [\"image\", \"text\"],\n",
+    "        \"index\": [0, 1],\n",
+    "        \"data\": [\n",
+    "            [\"\", \"text sample 1\"],  # the \"text\" column should contain empty string\n",
```

Comment:
Similar to the comment in cli examples. Can we use a dataset specific text here ?